10centos71_nettols/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos
+
+MAINTAINER Eugene Naryshkin <aurcame@gmail.com>
+
+RUN yum install -y net-tools 
+RUN yum install -y telnet
+
+ENTRYPOINT ["telnet"]
+
+CMD ["telnet 192.168.1.1 22"]
```

Comment:
CMD command is wrong. You've started to use an array syntax but within it added full shell command instead of separated arguments 1) IP address and 2) IP port